_posts/2017-12-18-new-blog.md

```diff
@@ -7,8 +7,10 @@ categories: [announcement, pinned]
 author: TheAssassin
 ---
 
+<img src="/img/new.png" class="postImg"/>
```

Comment:
There should probably be an empty line in between this line and the first paragraph of text.